crates/wash-cli/src/dev.rs

```diff
@@ -125,17 +229,80 @@ impl KnownDep {
             None => (iface, None),
             Some((iface, version)) => (iface, semver::Version::parse(version).ok()),
         };
-        match (iface, version) {
+        let (ns, pkg, iface, _) = parse_wit_meta_from_operation(format!("{iface}.none")).ok()?;
+        match (ns.as_str(), pkg.as_str(), iface.as_str()) {
+            // Skip explicitly ignored (normally internal) interfaces
+            (ns, pkg, iface) if is_ignored_iface_dep(ns, pkg, iface) => None,
             // Deal with known prefixes
-            ("wasi:keyvalue/atomics" | "wasi:keyvalue/store" | "wasi:keyvalue/batch", _) => {
-                Some(KnownDep::KeyValueProvider)
+            ("wasi", "keyvalue", "atomics" | "store" | "batch") => {
+                Some(Self::Receives(DependencySpecInner {
+                    wit: (
+                        ns,
+                        pkg,
+                        iface,
+                        version.map(VersionCoverage::SemVer).unwrap_or_default(),
+                    ),
+                    delegated_to_workspace: false,
+                    link_name: "default".into(),
+                    image_ref: Some(DEFAULT_KEYVALUE_PROVIDER_IMAGE.into()),
+                    // TODO: needs config on the source->target link (Bucket name)
+                    ..Default::default()
```

Comment:
Note for later, I think we also need to create the bucket too since the NATS KV provider depends on it

Reply:
Hmnn shouldn't that be done in the provider though? Did we remove bucket auto-creation?